packages/zoe/src/contractFacet/contractFacet.js

```diff
@@ -0,0 +1,348 @@
+// @ts-check
+
+// This is the Zoe contract facet. Each time we make a new instance of a
+// contract we will start by creating a new vat and running this code in it. In
+// order to install this code in a vat, Zoe needs to import a bundle containing
+// this code. We will eventually have an automated process, but for now, every
+// time this file is edited, the bundle must be manually rebuilt with
+// `yarn build-zcfBundle`.
+
+import { assert, details } from '@agoric/assert';
+import { E } from '@agoric/eventual-send';
+import makeWeakStore from '@agoric/weak-store';
+
+import makeAmountMath from '@agoric/ertp/src/amountMath';
+import { makeNotifierKit, updateFromNotifier } from '@agoric/notifier';
+import { makePromiseKit } from '@agoric/promise-kit';
+import { areRightsConserved } from './rightsConservation';
+import { makeIssuerTable } from '../issuerTable';
+import {
+  assertKeywordName,
+  getKeywords,
+  cleanProposal,
+} from '../zoeService/cleanProposal';
```

Comment:
This is the only cross-import between `src/contractFacet/` and `src/zoeService`. Aside from `cleanProposal`, all the other files they have in common are in neither directory. Typically, directly under `src`. Please move `cleanProposal` directly under `src/`.

```suggestion
import {
  assertKeywordName,
  getKeywords,
  cleanProposal,
} from '../cleanProposal';
```

I see now it was moved from there. Why?